contexts/credentials/v2

```diff
@@ -87,6 +87,69 @@
           "@container": "@graph"
         }
       }
+    },
+
+    "DataIntegrityProof": {
+      "@id": "https://w3id.org/security#DataIntegrityProof",
+      "@context": {
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "challenge": "https://w3id.org/security#challenge",
+        "created": {
+          "@id": "http://purl.org/dc/terms/created",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        },
+        "domain": "https://w3id.org/security#domain",
+        "expires": {
+          "@id": "https://w3id.org/security#expiration",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        },
+        "nonce": "https://w3id.org/security#nonce",
+        "proofPurpose": {
+          "@id": "https://w3id.org/security#proofPurpose",
+          "@type": "@vocab",
+          "@context": {
+            "@protected": true,
+            "id": "@id",
+            "type": "@type",
+            "assertionMethod": {
+              "@id": "https://w3id.org/security#assertionMethod",
+              "@type": "@id",
+              "@container": "@set"
+            },
+            "authentication": {
+              "@id": "https://w3id.org/security#authenticationMethod",
+              "@type": "@id",
+              "@container": "@set"
+            },
+            "capabilityInvocation": {
+              "@id": "https://w3id.org/security#capabilityInvocationMethod",
+              "@type": "@id",
+              "@container": "@set"
+            },
+            "capabilityDelegation": {
+              "@id": "https://w3id.org/security#capabilityDelegationMethod",
+              "@type": "@id",
+              "@container": "@set"
+            },
```

Comment:
```suggestion
```

Reply:
let's keep this out for now - might make sense, but is a dedicated topic on its own

Reply:
See: https://github.com/w3c/vc-data-model/pull/943#discussion_r993892902

Reply:
If we remove this, we deviate from what the Data Integrity spec's main context. We should have this discussion over there instead of over here.

In the meantime, we might want to think about what it means to not include the non-VC-ecosystem related things in the vc/v2 context. These are values that we expect to exist in `proofPurpose`, which is a Data Integrity thing... I'm not sure mixing and matching is the best approach, so we'd need to explore what happens if we decide to start mixing and matching (and the ways that could all go horribly wrong).

Reply:
Same comment as above, if we keep it in, we need to do a better job of explaining it.... that means adding informative text.

Reply:
Explanation and plan to move forward is here: https://github.com/w3c/vc-data-model/pull/943#issuecomment-1287491190